tests/test_ast.py

```diff
@@ -6,19 +6,31 @@
 def test_lite_repr():
     one = claripy.BVV(1, 8)
     two = claripy.BVV(2, 8)
-    a = claripy.BVS('a', 8)
-    b = claripy.BVS('b', 8)
+    a = claripy.BVS('a', 8, explicit_name=True)
+    b = claripy.BVS('b', 8, explicit_name=True)
 
-    nose.tools.assert_regex((a + one * b + two).shallow_repr(),
-                            r'\<BV8 __add__\(a_\d+_8, 1 \* b_\d+_8, 2\)\>')
-    nose.tools.assert_regex(((a + one) * (b + two)).shallow_repr(),
-                            r'\<BV8 \(a_\d+_8 \+ 1\) \* \(b_\d+_8 \+ 2\)\>')
-    nose.tools.assert_regex((a * one + b * two).shallow_repr(),
-                            r'\<BV8 a_\d+_8 \* 1 \+ b_\d+_8 \* 2\>')
-    nose.tools.assert_regex(((one + a)*(two + b)+(two + a)*(one + b)).shallow_repr(),
-                            r'\<BV8 \(1 \+ a_\d+_8\) \* \(2 \+ b_\d+_8\) \+ \(2 \+ a_\d+_8\) \* \(1 \+ b_\d+_8\)\>')
+    nose.tools.assert_equal((a + one * b + two).shallow_repr(),
+                            '<BV8 __add__(a, 1 * b, 2)>')
```

Comment:
can we make it so that this renders as `a + 1 * b + 2`?

Reply:
Yes. We can.

Reply:
The infix operator with more than 2 args is full represented. Even when using `details == Base.LITE_REPR`. I've changed this.